File: fizzbuzz2.rb
(1...200).each do |i|
  if (i % 3) == 0 && (i % 5) == 0
    puts "FizzBuzz"
  elsif (i % 3) == 0 && (i % 5) == 0 && (i % 7) == 0
    puts "FizzBuzzSivv"
  elsif (i % 3) == 0
    puts "Fizz"
  elsif (i % 5) == 0
    puts "Buzz"
  elsif (i % 7) == 0
    puts "Sivv"
  else
    puts i if (i % 3) != 0 && (i % 5) != 0
  end
end

File: fizzbuzz2.js
for (i = 1; i < 201; i++) {
  if ((i % 3) === 0 && (i % 5) === 0 && (i % 7) === 0) {
    console.log("FizzBuzzSivv");
  }
  else if ((i % 3) === 0 && (i % 5) === 0) {
    console.log("FizzBuzz");
  }
  else if (i % 3 === 0) {
    console.log("Fizz");
  }
  else if (i % 5 === 0) {
    console.log("Buzz");
  }
  else {
    console.log(i);
  }
}
